internal/character/serval/ult.go

```diff
@@ -0,0 +1,43 @@
+package serval
+
+import (
+	"github.com/simimpact/srsim/internal/global/common"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const Ult key.Attack = "serval-ult"
+
+func (c *char) Ult(target key.TargetID, state info.ActionState) {
+	if c.info.Eidolon >= 4 {
+		for _, trg := range c.engine.Enemies() {
+			c.engine.AddModifier(trg, info.Modifier{
```

Comment:
It only applies the modifier if the enemy doesnt have "Flag": "STAT_DOT_Electric"